src/app/modules/shared-components/components/page-navigator/page-navigator.component.ts

```diff
@@ -9,6 +11,8 @@ export class PageNavigatorComponent {
   @Input() allowWatching = false;
   @Input() isWatched = false;
   @Input() allowFullScreen = false;
+  @Input() startWatchGroupInfo?: StartWatchGroupInfo;
+  @Input() route?: GroupRoute;
```

Comment:
A "route" argument for a generic `PageNavigatorComponent` which is a `GroupRoute`... 
Ok, answer the previous questions first but probably this one will have to be fixed as well.

Reply:
So they will disappear with my previous remark.